lowfat/01-Gen-006-lowfat.xml

```diff
@@ -50,6 +52,8 @@
             morph="C"
             pos="conjunction"
             after="־"
+            gloss="(dm)"
+            transliteration="kiy-"
```

Comment:
@jonathanrobie Here's an example of where those attributes came through, which leads me to think I am looking in the right place, but it would be worth exploring why some of them apparently didn't make it into the Nodes trees (I can dig into this a bit in a few minutes). Perhaps we should open a separate issue for this.

Reply:
As a side note, "(dm)" does seem like an odd gloss. Perhaps 'discourse marker'?

Reply:
@jonathanrobie I think we should merge this. It is simply an update to the lowfat script to make sure all attributes in the Nodes trees are accounted for. If there are missing attributes in the Nodes trees themselves in some locations we should open a separate issue for that. No reason to not merge this update to the lowfat transform script though.